nano/core_test/conflicts.cpp

```diff
@@ -71,6 +71,79 @@ TEST (conflicts, DISABLED_add_two)
 	ASSERT_EQ (2, node1.active.size ());
 }
 
+TEST (conflicts, add_two)
+{
+	nano::system system{};
+	auto const & node = system.add_node ();
+
+	const auto do_send = [&node] (const auto & previous, const auto & from, const auto & to, bool forceConfirm = true)
+	-> std::pair<std::shared_ptr<nano::block>, std::shared_ptr<nano::block>> {
+		auto send = std::make_shared<nano::send_block> (previous->hash (), to.pub, 0, from.prv, from.pub, 0);
```

Comment:
If we are re-writing it, should we use a state block instead of of legacy block?

Reply:
Unfortunately that was not as trivial as it seems -- I tried for a few good hours hitting various hurdles and when everything seemed fine, I could not get passed the "work requirement". We can review it in the dev call if you want, but otherwise I'd leave it like this working rather than not working.